pkg/iac/rego/embed.go

```diff
@@ -66,10 +67,6 @@ func RegisterRegoRules(modules map[string]*ast.Module) {
 			continue
 		}
 
-		if !metadata.Deprecated {
-			regoCheckIDs.Append(metadata.AVDID)
-		}
```

Comment:
this is interesting, should we add a linter check to ensure a variable gets flagged if it's only set and never used?